### ISSUE TITLE
Is this repo implement learning rate decay in last 350 epochs?

### ISSUE THREAD

**Issue by Luciennnnnnn** [ohayonguy/PMRF#17]
Hi, I notice that you use a learning rate decay in last 350 epochs as mentioned in the appendix of your paper:

> In the last 350 epochs, we reduce the learning rate gradually, multiplying it by 0.98 at the end of every epoch

But, we do not see it in the code, Is this repo implement it? Or you found it do not improve performance now.

**Comment by ohayonguy** [comment]
The repo doesn’t implement it. Originally we had the lines of codes for this scheduling just commented out.
After 350 epochs, we stopped training. Then, we added the scheduler code, and resumed training from checkpoint. We used MultiStepLR from torch.

**Comment by Luciennnnnnn** [comment]
Thanks! It solved my question.
